packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/index.js

```diff
@@ -78,9 +81,13 @@ class BuySellContainer extends React.PureComponent {
 
     return (
       <Wrapper>
-        <Menu>
-          <Field name='status' component={TabMenuBuySellStatus} />
-        </Menu>
+        {
+          view.props.value.coinify.offline_token || view.props.value.sfox.account_token
```

Comment:
Can we use the canTrade function from services/ExchangeServce?

Reply:
I don't believe I made this change, I merged Phil W's branch into mine since I couldn't see buy.

Reply:
Ok, you need to merge development in again, and Ill make a note to phil